hack/windows/Install-OVS.ps1

```diff
@@ -199,6 +199,22 @@ function InstallOVS() {
     SetEnvVar "Path" $env:Path
 }
 
+function CheckIfLibInstalled() {
+    # Check if SSL library has been installed
+    $pathEntries = $env:Path.Split(";") | ForEach-Object {
```

Comment:
We can check the file existence in path `$OVSInstallDir\usr\sbin\` instead of looping in all the PATHs defined in `$env:PATH`?

Reply:
Actually I found when starting ovs service it will search SSL files in all paths of $env:PATH. So if related files exist in any path of $env:PATH, we don't need to download it anymore.

Reply:
OK.